plugins/out_s3/s3.c

```diff
@@ -294,6 +294,7 @@ static int read_seq_index(char *seq_index_file, uint64_t *seq_index)
 
     ret = fscanf(fp, "%"PRIu64, seq_index);
     if (ret != 1) {
+        fclose(fp);
```

Comment:
Its also needed a few lines below in write_seq_index?

Reply:
yup, thanks! Added